tests/factory/test_factory_invalid_options.py

```diff
@@ -2,14 +2,12 @@
 import os
 
 import mozfile
+import mozhttpd_base_test as mhttpd
```

Comment:
This is a local file so please move it back at the end of the imports.

Reply:
done

Reply:
Note that those tests have been changed to pytest, and this module is no longer in use. You will have to fix some merge conflicts when rebasing to master. Please do that.